var/spack/repos/builtin/packages/gobject-introspection/package.py

```diff
@@ -44,6 +45,12 @@ class GobjectIntrospection(MesonPackage, AutotoolsPackage):
     depends_on("glib@2.49.2:", when="@1.49.2:")
     # version 1.48.0 build fails with glib 2.49.4
     depends_on("glib@2.48.1", when="@1.48.0")
+    depends_on("glib@2.49.2:", when="@1.49.2:")
+    depends_on("glib@2.56:", when="@1.56")
+    depends_on("glib@2.72:", when="@1.72")
+    depends_on("glib@2.76:", when="@1.76")
+    depends_on("glib@2.79:", when="@1.79")
+
```

Comment:
These should be sorted like versions, that is, newest first. Also, do they have to be strict (`depends_on("glib@2.76", when="@1.76")`) or are newer glib versions ok?